### PR TITLE
Add basic enemy model

### PR DESCRIPTION
**Pull request by EPIC117**

### CODE REVIEW

File: VirusTowerDefense/Models/Enemy/BotAttack.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VirusTowerDefense.Models.Enemy
+{
+    class BotAttack
+    {
```
Comment (MichaelPuckett2) [inline VirusTowerDefense/Models/Enemy/BotAttack.cs]: Bot attack has no default values like Health and Speed.

File: VirusTowerDefense/Models/Enemy/PiratedGame.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace VirusTowerDefense.Models.Enemy
+{
+    public class PiratedGame: Enemy
+    {
+        public PiratedGame()
+        {
+            Health = 20;
+            Speed = .5;
```
Comment (MichaelPuckett2) [inline VirusTowerDefense/Models/Enemy/PiratedGame.cs]: all values we use will start with 0 if they are less than 1 like 0.5.

File: VirusTowerDefense/Models/Enemy/Spyware.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿namespace VirusTowerDefense.Models.Enemy
+{
+    public class Spyware : Enemy
+    {
+        public Spyware()
+        {
+            Health = 20;
+            Speed = .5;
```
Comment (MichaelPuckett2) [inline VirusTowerDefense/Models/Enemy/Spyware.cs]: add 0 before .5.

**Review by MichaelPuckett2 — CHANGES_REQUESTED** [review]
I have noticed a few things overlooked.

**Comment by EPIC117** [comment]
@MichaelPuckett2 I have made the changes mentioned in your review.

**Review by MichaelPuckett2 — APPROVED** [review]
Looks good!

**Pull request merged**